apps/vaporgui/PSliceSampleLocationSelector.h

```diff
@@ -0,0 +1,37 @@
+#pragma once
+
+#include "PGroup.h"
+#include "PLineItem.h"
+
+class VSliderEdit;
+
+//! \class PSliceSampleLocationSelector
+class PSliceSampleLocationSelector : public PGroup {
+public:
+    PSliceSampleLocationSelector();
+};
+
+class PSliceSampleLocationSelector1D : public PLineItem {
```

Comment:
Should this class be using a VDoubleSliderEdit?

Reply:
I guess so. I've replace it. Why do we still have `VSliderEdit` then?

Reply:
It just hasn't been replaced throughout the application.